### ISSUE TITLE
_Always up to date_ for private mirrors

### ISSUE THREAD

**Issue by brianjmurrell** [fedora-infra/mirrormanager2#338]
Per the [comment](https://github.com/fedora-infra/mirrormanager2/issues/304#issuecomment-998059509) in #304, there is an _Always up to date_ setting for public mirrors but that is not available for private mirrors.

It would be useful if that were available for private mirrors also.

**Comment by adrianreber** [comment]
Currently only a mirrormanager admin can set that flag. This is independent of public or private mirror.

**Comment by brianjmurrell** [comment]
Fair enough.  So an independent could request an admin to set that flag for them, yes?

**Comment by adrianreber** [comment]
> Fair enough. So an independent could request an admin to set that flag for them, yes?

Yes, that would be possible. Definitely more likely to happen then a quick code change.

**Comment by brianjmurrell** [comment]
But according to https://github.com/fedora-infra/mirrormanager2/issues/304#issuecomment-998067355 it's actually not implemented for private mirrors.  Or am I misunderstanding?

**Comment by adrianreber** [comment]
> But according to [#304 (comment)](https://github.com/fedora-infra/mirrormanager2/issues/304#issuecomment-998067355) it's actually not implemented for private mirrors. Or am I misunderstanding?

That is a misunderstanding. It is available for all mirrors, but only a mirrormanager admin can see and change it. We do not want people to turn it on because we have had problems with it in the past.

**Comment by brianjmurrell** [comment]
Ahhh.  I see.  How do I go about getting it turned on for my private mirror then?

**Comment by adrianreber** [comment]
Open a ticket at fedora-infrastructure as that is the place responsible for the Fedora MirrorManager instance: https://pagure.io/fedora-infrastructure/

**Comment by brianjmurrell** [comment]
Awesome, thanks!

But probably we can keep this issue opened for the coding work to make this something that users can do themselves?
